Documentation for grype integration tech preview

dspalmer99 opened the pull request:
Engine docs updates for grype integration tech preview

dspalmer99 commented on docs/content/docs/grype/_index.md:
```diff
@@ -0,0 +1,46 @@
+both vulnerability scanners at the same time. This configuration is picked up at bootstrap, and cannot be changed ona running system. 
+
+#### Running with docker-compose
+1. Install or update to Anchore Engine 0.10.0.
```
I'd like to get confirmation that these configuration options are correct, particularly for helm as I've not run it that way yet. (I based this off of what I saw in your charts PR, @Btodhunter

Vijay-P replied:
this is right. the other variable is ANCHORE_GRYPE_DB_URL, but that only needs to be configured differently for enterprise

zhill commented on docs/content/docs/grype/_index.md:
```diff
@@ -0,0 +1,46 @@
+
+#### Running with helm
+1. Install or update to Anchore Engine 0.10.0.
+2. Update the following value in `stable/anchore-engine/values.yaml`:
```
Users won't use the git repo to get the chart. They'll create their own values.yaml. I think here we should link to the chart README on how to set this since the specifics are part of the chart

zhill commented on docs/content/docs/grype/_index.md:
```diff
@@ -0,0 +1,46 @@
+    anchorePolicyEngine
+      ...
+      vulnerabilityProvider: grype
+3. Redeploy the services
```
`helm upgrade`

zhill reviewed (changes requested):
Suggestions for chart/helm instructions

Vijay-P commented on docs/content/docs/grype/_index.md:
```diff
@@ -0,0 +1,46 @@
+3. Redeploy the services
+
+After making the relevant change above and redeploying, the system will start up with the grype vulnerability scanner enabled and will
+sync the latest version of grype db. Note that legacy feeds will no longer be synced while grype is configured. All vulnerability data
```
We should provide an example configuration here.

Vijay-P commented on docs/content/docs/install/configuration/feed_configuration.md:
````diff
@@ -43,6 +43,11 @@ feeds:
 ```
 
+***Note:*** Engine now has a new feed for Grype. This feed will only be synced if the
+[tech preview Grype vulnerability scanner]({{< ref "/docs/grype/_index.md" >}}) is enabled. Furthermore, if it is
````
hyperlink is broken. check that you're using the right MD syntax

Vijay-P replied:
the example that i mentioned before can also be put here.

Vijay-P commented on docs/content/docs/install/configuration/feed_configuration.md:
````diff
@@ -43,6 +43,11 @@ feeds:
   read_timeout_seconds: 60
 ```
 
+***Note:*** Engine now has a new feed for Grype. This feed will only be synced if the
````
the old style config was moved to a new location. need to update the examples here to reflect that location. `services.policy_engine.vulnerabilities.sync.data`

Vijay-P reviewed (changes requested):
great work! Config changes introduced with grype integration need to be documented.

dspalmer99 commented:
@zhill @Vijay-P I've addressed all feedback, and fixed some additional typos. Can I get another look from you both on this. Swathi's PR has merge priority, but I'd like to get this one in today as well.

zhill reviewed (approved):
There are some caveats we need to add for unsupported functionality in this mode, but I think that can be added as a follow-up to this either in release notes directly or embedded in this page.
